lib/plugins/chat.js

```diff
@@ -84,6 +84,7 @@ function inject (bot, options) {
   })
 
   function chatWithHeader (header, message) {
+    if (typeof message === 'number') message = message.toString()
     if (typeof message !== 'string') {
       throw new Error('Incorrect type! Should be a string.')
```

Comment:
Probably change the error message to: 'Incorrect type! Should be a string or number.'